external_libraries/a_matrix/include/matrix.h

```diff
@@ -263,4 +273,20 @@ inline std::ostream& operator<<(std::ostream& rOStream,
     return rOStream;
 }
 
+template <typename TExpressionType, std::size_t TCategory>
+inline std::ostream& operator<<(std::ostream& rOStream,
```

Comment:
it works if I comment this function

Reply:
oops, seems that one commit is missing here. You may comment this function safely. 

We have another version of this function in Kratos which mimics the ublas output....